internal/gapicgen/generator/genproto.go

```diff
@@ -139,6 +139,8 @@ func (g *GenprotoGenerator) Regen(ctx context.Context) error {
 				log.Println("running protoc on", pk)
 				return g.protoc(fn, grpc)
 			})
+		} else {
+			log.Panicf("skipping, %q has been migrated", pkg)
```

Comment:
Why panic here?

Reply:
whoa, that was a mistake. Auto-complete failed me there. I will update rn